frr-tools/status/exporter.go

```diff
@@ -0,0 +1,116 @@
+// SPDX-License-Identifier:Apache-2.0
```

Comment:
nit: we took this (frr-tools) from metallb, because it was effectlvely a subset of tools. Here it's more or less all frr, so it'd make sense to have a cmd/ for all the binaries.
Let's file an issue so we can address later.